core/runtime/jsrealm.rs

```diff
@@ -48,12 +48,12 @@ impl Hasher for IdentityHasher {
   feature = "op_driver_joinset",
   not(feature = "op_driver_futuresunordered")
 ))]
-type DefaultOpDriver = super::op_driver::JoinSetDriver;
+pub(crate) type RealOpDriver = super::op_driver::JoinSetDriver;
 
 #[cfg(feature = "op_driver_futuresunordered")]
-type DefaultOpDriver = super::op_driver::FuturesUnorderedDriver;
+pub(crate) type RealOpDriver = super::op_driver::FuturesUnorderedDriver;
 
-pub(crate) struct ContextState<OpDriverImpl: OpDriver = DefaultOpDriver> {
+pub(crate) struct ContextState<OpDriverImpl: OpDriver = RealOpDriver> {
```

Comment:
Remove this generic and just use `OpDriverImpl`. This particular generic was more important during the testing phase, but we never actually need to make `ContextState` generic.